src/components/Trivia.js

```diff
@@ -142,20 +144,23 @@ class Trivia extends Component {
         >
           {`Question : ${question.question}`}
         </h2>
-        { options.map((answer, key) => (
-          <button
-            type="button"
-            data-testid={ answer === question.correct_answer
-              ? correct : `wrong-answer-${key}` }
-            className={ answer === question.correct_answer
-              ? correct : `wrong-answer-${key}` }
-            onClick={ (e) => this.handleCickInAnswer(e) }
-            disabled={ timeOut }
-            key={ key }
-          >
-            { answer }
-          </button>
-        ))}
+        { options.map((answer) => {
+          if (answer !== question.correct_answer) { answerIndex += 1; }
+          return (
+            <button
+              type="button"
+              data-testid={ answer === question.correct_answer
+                ? correct : `wrong-answer-${answerIndex}` }
+              className={ answer === question.correct_answer
+                ? correct : `wrong-answer-${answerIndex}` }
+              onClick={ (e) => this.handleCickInAnswer(e) }
+              disabled={ timeOut }
+              key={ answerIndex }
```

Comment:
```suggestion
              key={ if (answer === question.correct_answer) return 4; }
```
A ideia é colocar uma key diferente para quando a resposta for a correta.